prow/jobs/jinja/presubmits/unit_test.jinja2

```diff
@@ -0,0 +1,17 @@
+{% for service in aws_services  %}
+  aws-controllers-k8s/{{ service }}-controller:
+  - name: {{ service }}-unit-test
+    decorate: true
+    spec:
+      serviceAccountName: pre-submit-service-account
+      containers:
+      - image: public.ecr.aws/aws-controllers-k8s/prow:prow-unit-0.0.1
```

Comment:
I guess now [`jobs/jinja/presubmits/e2e_test.jinja2`](https://github.com/aws-controllers-k8s/test-infra/blob/main/prow/jobs/jinja/presubmits/e2e_test.jinja2#L20) should use the integration image?  something like `.../prow:prow-integration-0.0.1`

Reply:
Yeah I will push a new version of this into the repo just for the sake of renaming it.